src/ui.rs

```diff
@@ -667,110 +669,198 @@ impl PofToolsGui {
+                                        }
+                                        Warning::SpecialPointNameTooLong(idx) => {
+                                            format!("Special point name '{}'", self.model.special_points[*idx].name)
+                                        }
+                                        Warning::DockingBayNameTooLong(idx) => {
+                                            format!("Docking bay name '{}'", self.model.special_points[*idx].name)
+                                        }
+                                        _ => unreachable!(),
+                                    };
+                                    format!("⚠ {} is too long (max {} bytes)", field, pof::MAX_NAME_LEN - 1)
+                                }
+                                Warning::GlowBankPropertiesTooLong(_)
+                                | Warning::ThrusterPropertiesTooLong(_)
+                                | Warning::SubObjectPropertiesTooLong(_)
+                                | Warning::DockingBayPropertiesTooLong(_)
+                                | Warning::SpecialPointPropertiesTooLong(_) => {
+                                    let field = match warning {
+                                        Warning::GlowBankPropertiesTooLong(idx) => {
+                                            format!(
+                                                "Glow bank {} ({}) properties",
+                                                *idx,
+                                                pof::properties_get_field(&self.model.glow_banks[*idx].properties, "$glow_texture")
+                                                    .unwrap_or_default()
+                                            )
+                                        }
+                                        Warning::ThrusterPropertiesTooLong(idx) => {
+                                            format!("Thruster bank {} properties", *idx + 1)
+                                        }
+                                        Warning::SubObjectPropertiesTooLong(id) => {
+                                            format!("Subobject {} properties", self.model.sub_objects[*id].name)
+                                        }
+                                        Warning::DockingBayPropertiesTooLong(idx) => {
+                                            format!("Docking bay {} properties", *idx + 1)
+                                        }
+                                        Warning::SpecialPointPropertiesTooLong(idx) => {
+                                            format!("Special point {} properties", self.model.special_points[*idx].name)
+                                        }
+                                        _ => unreachable!(),
+                                    };
+                                    format!("⚠ {} is too long (max {} bytes)", field, pof::MAX_PROPERTIES_LEN)
```

Comment:
```suggestion
                                    format!("⚠ {} is too long (max {} bytes)", field, pof::MAX_PROPERTIES_LEN - 1)
```
Better yet, make `MAX_PROPERTIES_LEN` be 255 instead of 256 and adjust elsewhere as necessary, same for `MAX_NAME_LEN`